fhir-persistence/src/test/resources/logging.unitTest.properties

```diff
@@ -49,5 +49,6 @@ java.util.logging.ConsoleHandler.formatter = java.util.logging.SimpleFormatter
 ############################################################
```

Comment:
Update copyright date

Reply:
lol, this one has a beginning copyright of 2000... should we change that to 2016 when our project started?

Reply:
Sounds good to me.